app/controllers/update-job-view-model.js

```diff
@@ -1,19 +1,34 @@
 const progression = require('../models/progression');
 const progressionDecorator = require('./progression-view-decorator');
+const ratings = require('../models/ratings');
+const moment = require('moment');
+const DATE_FORMAT = 'YYYY-MM-DD';
 
 class UpdateJobViewModel {
-  constructor(accountId, job, basePath, csrfToken) {
+  constructor(accountId, job, basePath, csrfToken, validationErrors) {
     Object.assign(this, job);
     this.csrfToken = csrfToken;
     this.accountId = accountId;
     this.basePath = basePath;
+    this.errors = validationErrors;
 
-    const ratingOptionList = [5, 4, 3, 2, 1].map(value => ({ value }));
-    this.ratingOptions = this.addCheckedFlag(ratingOptionList, job.rating);
+    this.ratingOptions = this.addCheckedFlag(
+      ratings().reverse().map(value => ({ value })),
+      job.rating
+    );
+
```

Comment:
If the View Model is now responsible for the translation of DB objects to view layer formats should the DB Model be responsible for translation of DB objects from view to DB.  e.g. also move:
`
  ['deadlineDate', 'applicationDate', 'interviewDate']
    .forEach(df => { if (updateData[df] === '') { delete updateData[df]; } });
`
to jobs-model